### PR TITLE
Draft of example Health-RI core dataset notebook

### PR DESCRIPTION
**Pull request by Markus92**
This pull request updates both Health-RI core models and creates an example on how to use the models to generate Health-RI compliant DCAT metadata.
The metadata could then be pushed to a FAIR Data Point, but that is outside the scope of this pull request.

### CODE REVIEW

File: docs/Usage_example_HealthRI_core.ipynb [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+    "\n",
+    "# Description can stay as is, but we put it in a List (reason: i18n support).\n",
+    "df[\"description\"] = df[\"description\"].apply(lambda x: [x])\n",
+    "\n",
+    "# Name is called \"title\" in dcat. We first put it in a List, then rename it\n",
+    "df[\"name\"] = df[\"name\"].apply(lambda x: [x])\n",
+    "df.rename(columns={\"name\": \"title\"}, inplace=True)\n",
+    "\n",
+    "# creator is a foaf:Agent. We add the mandatory tags for it\n",
+    "df[\"creator\"] = df.apply(lambda x: [{'name': [\"author_name\"], x[\"author_id\"]], axis=1)\n",
```
Comment (kburger) [inline docs/Usage_example_HealthRI_core.ipynb:137]: `SyntaxError: closing parenthesis ']' does not match opening parenthesis '{'`. I'm not too familiar with pandas, so purely on syntax should it be sth like 

```suggestion
    "df[\"creator\"] = df.apply(lambda x: [{'name': x[\"author_name\"]}, 'identifier': x[\"author_id\"]], axis=1)\n",
```
❓

**Review by kburger — CHANGES_REQUESTED** [review]
Running it locally caused some issues.

File: docs/Usage_example_HealthRI_core.ipynb [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+      " 'publisher': {'RDF type': 'uri', 'datatype': 'List[Union[Url, Agent]]'},\n",
+      " 'theme': {'RDF type': 'uri', 'datatype': 'List[Url]'},\n",
+      " 'title': {'RDF type': 'rdfs_literal', 'datatype': 'List[LiteralField]'}}\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "c:\\Users\\Mark\\AppData\\Local\\hatch\\env\\virtual\\sempyro\\DMlT67Hc\\docs\\Lib\\site-packages\\pydantic\\_internal\\_generate_schema.py:355: UserWarning: [typing.Annotated[pydantic_core._pydantic_core.Url, UrlConstraints(max_length=None, allowed_schemes=['http', 'https'], host_required=None, default_host=None, default_port=None, default_path=None)]] is not a Python type (it may be an instance of an object), Pydantic will allow any object with no validation since we cannot even enforce that the input is an instance of the given type. To get rid of this error wrap the type with `pydantic.SkipValidation`.\n",
```
[the remark this answers is not in the document]
Comment (Markus92) [reply docs/Usage_example_HealthRI_core.ipynb]: It's a warning in SeMPyRO I'm afraid...

File: docs/Usage_example_HealthRI_core.ipynb (hunk 1 above)
Comment (Markus92) [reply docs/Usage_example_HealthRI_core.ipynb]: Thanks for spotting this!

**Review by kburger — APPROVED** [review]
LGTM

**Pull request merged**